lib/registries/addon/overview/route.ts

```diff
@@ -24,6 +71,10 @@ export default class Overview extends GuidRoute {
         };
     }
 
+    afterModel(this: Overview, model: any) {
```

Comment:
You don't need to give `this` a type for class methods.

Also, try to avoid using `any`. In this case you can do `model: GuidRouteModel<Registration>`.

Reply:
If I omit `this`, Typescript linter cannot figure the type of `this.get('setHeadTags')` of and hence does not allow doing `.perform()`.

Reply:
Ah, I see. The task should be defined on the prototype, so you don't have to use `get()`:
```ts
class Overview extends GuidRoute.extend({
  setHeadTags: task(...),
}) {
  // ...
  afterModel(model: GuidRouteModel<Registration>) {
    this.setHeadTags.perform(model);
  }
}
```
We should probably stick to that pattern until we upgrade to using decorators for tasks.